### DISCUSSION TITLE
Table 26, Panel 2

### DISCUSSION THREAD

**Discussion by stefanhenneking** [cssi-pi-community/2022-meeting#47] [Panel 2 roundtable (Monday afternoon)]
Table 26, Panel 2 discussion

**Comment by terletska** [comment]
Data management should be a more rigorous component of the proposal, so the PI and their institutions to be proactive with the data.
Data and software should be taken as "first-class citizens" similarly to the published papers.

**Comment by terletska** [comment]
Also, visibility of the available resources should be better communicated to the PI to utilize better the usage of the NSF-funded resources.

**Reply by vjdorazio** [reply]
especially resources that do not require a formal proposal, or have a quick turnaround

**Comment by vjdorazio** [comment]
discussed software as a more central component of nsf data management plan
